src/utils/common.js

```diff
@@ -595,3 +595,8 @@ export const formatBigFiatAmount = (value: Value, fiatCurrency: string) => {
 export const getEnsName = (username: string) => `${username}${getEnsPrefix()}`;
 
 export const extractUsernameFromEnsName = (ensName: string) => ensName.replace(getEnsPrefix(), '');
+
+export const valueAsKey = (value: string | number): string => {
+  if (!value) return '';
+  return value.toString().toLowerCase();
+};
```

Comment:
I would also add 
```
export const valueForAddress = <Value>(record: Record<Value>, address: ?string): ?Value => {
   return record[valueAsKey(address)];
}
```

Which would slightly improve call site by reducing parenthesis nesting & correctly indicate that the key might return nil value.

```
if (!balances[valueAsKey(gasTokenAddress)]) return false;
```
 =>
```
if (!valueForAddress(balances, gasTokenAddress)) return false;
```